hera_pspec/pspecdata.py

```diff
@@ -2186,9 +2268,16 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
             given an input visibility noise model, and store the output
             in the UVPSpec object.
 
+        cov_model : string, optional
+            specifies how covariance are to be calculated. Currently supports
+            dsets and empirical
+
```

Comment:
It may be helpful here to briefly describe what each of them do, and what is the default choice.

Reply:
I've extended these doc-strings.